### PR TITLE
quality fixes including installation of six, docstring stuff, and pii…

### PR DESCRIPTION
**Pull request by christopappas**
…_check suppression

### CODE REVIEW

File: edx_rbac/models.py [hunk 1]
```diff
@@ -9,18 +9,22 @@
...
 class UserRoleAssignmentCreator(ModelBase):
     """
     The model extending UserRoleAssignment should get a foreign key to a model that is a subclass of UserRole.
     """
 
-    def __new__(cls, name, bases, attrs):
-        model = super(UserRoleAssignmentCreator, cls).__new__(cls, name, bases, attrs)
+    def __new__(mcs, name, bases, attrs):
+        """
+        Override to dynamically create foreign key for objects begotten from abstract class.
```
Comment (christopappas) [inline edx_rbac/models.py]: Happy changing this comment to something else @brittneyexline

File: edx_rbac/models.py [hunk 2]
```diff
@@ -55,15 +64,19 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
-class UserRoleAssignment(TimeStampedModel):
+class UserRoleAssignment(with_metaclass(UserRoleAssignmentCreator, TimeStampedModel)):
```
Comment (brittneyexline) [inline edx_rbac/models.py]: is it intentional to include TimeStampedModel in this with_metaclass call?

Comment (christopappas) [reply edx_rbac/models.py]: hmm good eye. i think i misread the documentation with this. specifically, what the 2nd argument of that should be. will give it another pass

Comment (christopappas) [reply edx_rbac/models.py]: i think this is right?
```
Create a new class with base class base and metaclass metaclass. This is designed to be used in class declarations like this:

from six import with_metaclass

class Meta(type):
    pass

class Base(object):
    pass

class MyClass(with_metaclass(Meta, Base)):
    pass```

**Pull request merged**